chat/coyim/Makefile

```diff
@@ -7,8 +7,8 @@ GITHUB_TAG=	v${PKGVERSION_NOREV}
 EXTRACT_USING=	bsdtar
 
 MAINTAINER=	fox@NetBSD.org
-HOMEPAGE=	https://github.com/coyim/coyim
-COMMENT=	Safe and secure chat client
+HOMEPAGE=	https://coy.im/
+COMMENT=	A safe and secure by default chat client
```

Comment:
Due to the rules about COMMENT in pkgsrc, a comment should not start with "A" / "An" or "The".

Reply:
Changed ;)

Reply:
LGTM, pulling in the changes.

Thank you.